P1/Project_Code/frontend/src/routes/Register/Register.route.jsx

```diff
@@ -57,6 +49,11 @@ function Register() {
 	const validateInput = async (data) => {
 		let inputErrors = {};
 
+		if (!isEmail(data.email))
```

Comment:
Same as my last comment add an explicit check that data.email values are not undefined or null.